Dockerfile

```diff
@@ -1,29 +1,59 @@
-FROM opendatacube/datacube-core:1.7
+FROM opendatacube/geobase:wheels as env_builder
 
-# Install the heavy stuff first
-RUN apt-get update && apt-get install -y gfortran \
-    && rm -rf /var/lib/apt/lists/* \
-    && pip3 install git+https://github.com/GeoscienceAustralia/fc --no-deps --global-option=build --global-option='--executable=/usr/bin/env python3'  \
-    && rm -rf $HOME/.cache/pip
+COPY requirements.txt /
+RUN env-build-tool new /requirements.txt /env
 
-RUN pip3 install git+https://github.com/GeoscienceAustralia/wofs --no-deps --global-option=build --global-option='--executable=/usr/bin/env python3'
+ENV PATH=${py_env_path}/bin:$PATH \
+    PYTHONPATH=${py_env_path}
```

Comment:
this doesn't look right `PYTHONPATH`. Why you need to set it at all, and if you do it's not this, it will need some extra paths below the env path.

Reply:
Ah, I copied from the OWS image, here: https://github.com/opendatacube/datacube-ows/blob/master/Dockerfile#L12

@tom-butler we probably want to remove it there too

Reply:
no it has to be in ows as datacube-ows-update expects the config file to be accessible in the python path. Shouldn't be in other repos though